k8s/helmfile/helmfile.yaml

```diff
@@ -75,8 +82,7 @@ releases:
     installed: {{ ne .Environment.Name "local" | toYaml }}
     namespace: default
     chart: ./../../charts/certificates
-    values:
-      - "env/{{ .Environment.Name }}/certificates.values.yaml.gotmpl"
+    <<: *default_release
```

Comment:
The documentation says `inherit` should exist, but at least on my local setup, it does not work and I will see the following error:

```
in ./helmfile.yaml: failed to read helmfile.yaml: reading document at index 1: yaml: unmarshal errors:
  line 96: field inherit not found in type state.ReleaseSpec
```

Reply:
that's good to know - I think the YAML anchors are fine for now. If you want feel free to leave a comment in the template section for this so we have a hint to use inheritance in the future